core/src/main/java/org/lflang/diagram/synthesis/LinguaFrancaSynthesis.java

```diff
@@ -292,6 +292,7 @@ public List<SynthesisOption> getDisplayedSynthesisOptions() {
         ModeDiagrams.INITIALLY_COLLAPSE_MODES,
         SHOW_USER_LABELS,
         SHOW_HYPERLINKS,
+        LinguaFrancaStyleExtensions.SELECTION_HIGHLIGHTING_COLOR,
```

Comment:
I ordered the selection highlighting option below the more visual synthesis options. Feel free to comment on this